app/scripts/patient-dashboard/directives/patient-relationship.directive.js

```diff
@@ -0,0 +1,228 @@
+(function(){
+'use strict';
+angular
+      .module('app.patientdashboard')
+      .directive('patientRelationship',patientRelationship);
+function patientRelationship(){
+  var patientRelationshipDefinition = {
+      restrict: 'EA',
+      templateUrl: 'views/patient-dashboard/patient-relationships-template.html',
+      controller: PatientRelationshipCtrl
+  };
+
+  return patientRelationshipDefinition;
+}
+PatientRelationshipCtrl.$inject = ['$rootScope', '$scope', '$stateParams', 'OpenmrsRestService','$state','dialogs', '$timeout'];
+function PatientRelationshipCtrl($rootScope, $scope, $stateParams, OpenmrsRestService,$state,dialogs, $timeout){
+  $scope.patientRelationships = undefined;
+  $scope.showSuccessAlert = false;
+  $scope.successAlert='';
+  var patientRelationship = getPatientRelationships();
+  var patientRelationshipType=OpenmrsRestService.getPatientRelationshipTypeService()
+  .getPatientRelationshipTypes(function(data){
+    $scope.patientRelationshipTypes=data;
+  },function(error){
+    console.log("The request failed because of ",error);
+  });
+  $scope.loadPatient = loadPatient;
+  function loadPatient(patientUuid) {
+    /*
+      Get the selected patient and save the details in the root scope
+      so that we don't do another round trip to get the patient details
+      */
+      console.log("patient uuid clicked id ",patientUuid);
+    OpenmrsRestService.getPatientService().getPatientByUuid({ uuid: patientUuid },
+           function(data) {
+             $rootScope.broadcastPatient = data;
+             $state.go('patient', { uuid: patientUuid });
+           }
+      );
+  }
+  $scope.setPatientRelationship=setPatientRelationship;
+  function setPatientRelationship(personA,relationshipTypeUUid,personB){
```

Comment:
Add one line between functions to make it readable. Also this function is long and there exists opportunities to reuse bits of your code. For e.g. you could take the success and error callbacks to their own functions, and avoid the anonymous functions.